### PR TITLE
Accessibility page

### PR DESCRIPTION
**Pull request by scottcwilson**
Provides a page for storeowners to make an accessibility statement about their site, which some lawyers are now recommending.

### CODE REVIEW

File: includes/templates/template_default/templates/tpl_accessibility_default.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * @copyright Copyright 2003-2022 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: DrByte 2020 Jul 10 Modified in v1.5.8-alpha $
+ */
+?>
+<div class="centerColumn" id="accessibility">
+<h1 id="accessibilityDefaultHeading"><?php echo HEADING_TITLE; ?></h1>
+
+<div id="accessibilityprivacyDefaultMainContent" class="content">
```
Comment (drbyte) [inline includes/templates/template_default/templates/tpl_accessibility_default.php:14]: Probably don't need the word `privacy` in here.

File: includes/languages/english/lang.accessibility.php [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?php
+$define = [
+    'NAVBAR_TITLE' => 'Accessibility Statement',
+    'HEADING_TITLE' => 'Accessibility Statement',
+    'TEXT_INFORMATION' => 'Your Accessibility Statement should be on this page.',
```
Comment (drbyte) [inline includes/languages/english/lang.accessibility.php:5]: This `TEXT_INFORMATION` line is never used. 
Probably ideal to remove it, in favor of the storeowner updating the actual page content via the Define Pages Editor.

**Comment by scottcwilson** [comment]
Fixed both issues.

**Comment by scottcwilson** [comment]
This has been submitted as a plugin for 1.5.8 and below.

**Comment by scottcwilson** [comment]
One small issue was identified and fixed in #6280.

**Pull request merged**